src/http/gateway/resources/gateway.js

```diff
@@ -12,6 +12,21 @@ const Stream = require('readable-stream')
 const { resolver } = require('ipfs-http-response')
 const PathUtils = require('../utils/path')
 
+function detectContenType (ref, chunk) {
```

Comment:
Minor typo here `detectContenType` - missing "t" on "conten"

Reply:
Will fix once the workshop I’m attending on dweb ends. :p